penguin/web/services/token_usage.py

```
@@ -0,0 +1,26 @@
+"""Token usage service helpers for route handlers."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from fastapi import HTTPException
+
+
+def get_token_usage_payload(
+    core: Any,
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `core`

(ANN401)